Week3/fuel.py

```diff
@@ -0,0 +1,22 @@
+
+def main():
+    gauge = fuel_tank()
+    if gauge <= 1:
+        print("E")
+    elif gauge >= 99:
+        print("F")
+    else:
+        print(gauge,"%")
+
+def fuel_tank():
+    while True:
+        fuel = input("Fraction: ")
+        try:
+            numerator, denominator = fuel.split("/")
+            f = int(numerator) / int(denominator)
```

Comment:
at this point, `f` if an `int` value because `int(numerator)` and `int(denominator)` turned `string` values into integers, but  `numerator` and `denominator` are still remaining string values,  then integer divided by integer resulted in integer `f`.